dev_notes/past_release_notes/v0.6.0.md

```diff
@@ -0,0 +1,28 @@
+# MongoDB Kubernetes Operator 0.6.0
+## Kubernetes Operator
+
+* Breaking Changes
+  * A new VolumeClaimTemplate has been added `logs-volume`. When you deploy the operator, if there is an existing StatefulSet the operator will attempt to perform an invalid update. The existing StatefulSet must be deleted before upgrading the operator.
+  
+  * The user of the mongod and mongodb-agent containers has changed. This means that there will be permissions
+    issues when upgrading from an earlier version of the operator. In order to update the permissions in the volume, you can use an init container.
+
+* Upgrade instructions
+
+  Remove the current operator deployment
+  -  `kubectl delete deployment <operator-deployment>`
+  Delete the existing StatefulSet for the MongoDBCommunity resource
+  Note: to ensure existing data is not lost, ensure that the retain policy of your Persistent Volumes is configured correctly. 
```

Comment:
nit: I think it we should mention here how to verify this since it's a crucial step.